canonical-web-and-design/jaas-dashboard
# Render the controller name in the model table

## Done
Render the controller name in the model table

## QA
- `juju bootstrap` into a public cloud
- `juju dashboard` and connect to that dashboard to accept the self-signed cert.
  - take note of the ip address and u/p
- Follow the deployment steps in #526 
- Open config.js and change: 
  - `baseControllerURL` to the ip address and port (if any) from above.
  - `isJuju` to `true`
  - `identityProviderAvailable` to `false`
- `./run`
- Open http://0.0.0.0:8036/
- You should see your controller name in the list instead of the uuid

## Details
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/532

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/83283660-7917dd80-a1d3-11ea-981c-02514427b96c.png)
